api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
```

Comment:
I've seen a lot of passing in of empty dictionaries to this function. Would it make more sense to have default parameters of {} for each of these and then just not passing parameters in if they're empty?